rpcs3/Emu/Cell/Modules/cellAudioOut.cpp

```diff
@@ -434,7 +404,7 @@ error_code cellAudioOutConfigure(u32 audioOut, vm::ptr<CellAudioOutConfiguration
 		}
 	}
 
-	cellSysutil.notice("cellAudioOutConfigure: channels=%d, encoder=%d, downMixer=%d", config->channel, config->encoder, config->downMixer);
+	cellSysutil.notice("Selected audio configuration: channels=%d, encoder=%d, downMixer=%d", config->channel, config->encoder, config->downMixer);
```

Comment:
it's supposed to mention the function

Reply:
Oh I just copied that from cellVideoOut, then that one was wrong